docs/v1/P4Runtime-Spec.mdk

```diff
@@ -2369,13 +2369,17 @@ whose underlying data plane representation may vary on different targets, but
 for which it may be convenient to present a unified representation and numbering
 scheme to the control plane. **The `@p4runtime_translation` annotation can only
 be used if the underlying P4 built-in type is a fixed-width unsigned bitstring
-type (`bit<W>`)** and the type exposed to the control plane will also be a
-fixed-width unsigned bitstring, with a potentially different bitwidth. It takes
-two parameters: a *URI* (Uniform Resource Identifier) which uniquely identifies
-the translation being performed on entities of the new type to the P4Runtime
-server and the *bitwidth* of the bitstring type exposed to the control plane. It
-is recommended that the URI includes at least the P4 architecture name and the
-type name.
+type (`bit<W>`)** . The type exposed to the control plane can be either a
+fixed-width unsigned bitstring, with a potentially different bitwidth, or a
+string. The annotation takes two parameters: a *URI* (Uniform Resource
+Identifier) which uniquely identifies the translation being performed on
+entities of the new type to the P4Runtime server and the *controller_type* of
+the values exposed the control plane. The *controller_type* can be either
```

Comment:
maybe insert "to", e.g. "values exposed to the control plane"?

Reply:
done